### PR TITLE
[blog] CFD 01

### PR DESCRIPTION
**Pull request by Olinaaaloompa**
### Purpose

- No issue is linked to this PR.

### Changes

- No changes.

### Review Instructions

- No instructions.

### CODE REVIEW

File: blogs/en/2022/09/15/accelerate-pytorch.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ authors:
...
+Our previous blogs (Taichi & PyTorch [01](//blogs/en/2022/08/08/Taichi_and_Torch_01.md) and [02](//blogs/en/2022/08/15/Taichi_and_Torch_02.md)) pointed out that Taichi and Torch serve different application scenarios: Taichi can provide finer control over parallelization and enables more 'granular' (element-level) operations, giving its users much more flexibilities; Torch abstracts such details into Tensor-level operations like LEGO bricks, enabling its users to focus on building Machine Learning (ML) models. Taichi is one of the most sought-after frameworks in Computer Graphics, so is Torch in ML. The question is: can they complement each other? And the answer is an unequivocal yes! In this blog, we will use two simple examples to explain how to use Taichi kernel to implement data preprocessing operators or custom ML operators. With Taichi, you can accelerate your ML model development with ease and get rid of the tedious low-level parallel programming (CUDA for example) for good.
```
Comment (Vissidarte-Herman) [inline blogs/en/2022/09/15/accelerate-pytorch.md:11]: incorrect link format.

Comment (Vissidarte-Herman) [reply blogs/en/2022/09/15/accelerate-pytorch.md:11]: You can use full url here.

**Review by Vissidarte-Herman — CHANGES_REQUESTED** [review]

File: blogs/en/2022/09/15/accelerate-pytorch.md (hunk 1 above)
Comment (Olinaaaloompa) [inline blogs/en/2022/09/15/accelerate-pytorch.md:11]:
```suggestion
Our previous blogs (Taichi & PyTorch [01](https://docs.taichi-lang.org/blog/taichi_vs_torch_01) and [02](https://docs.taichi-lang.org/blog/taichi_vs_torch_02)) pointed out that Taichi and Torch serve different application scenarios: Taichi can provide finer control over parallelization and enables more 'granular' (element-level) operations, giving its users much more flexibilities; Torch abstracts such details into Tensor-level operations like LEGO bricks, enabling its users to focus on building Machine Learning (ML) models. Taichi is one of the most sought-after frameworks in Computer Graphics, so is Torch in ML. The question is: can they complement each other? And the answer is an unequivocal yes! In this blog, we will use two simple examples to explain how to use Taichi kernel to implement data preprocessing operators or custom ML operators. With Taichi, you can accelerate your ML model development with ease and get rid of the tedious low-level parallel programming (CUDA for example) for good.
```

Comment (Olinaaaloompa) [reply blogs/en/2022/09/15/accelerate-pytorch.md]: Have replaced the links.

**Review by Vissidarte-Herman — APPROVED** [review]
LGTM

**Pull request merged**